lib/puppet/face/module/upgrade.rb

```diff
@@ -67,7 +67,7 @@
     end
 
     when_invoked do |name, options|
-      name = name.gsub('/', '-')
+      name = name.tr('/', '-')
```

Comment:
You could use `name.tr!`

Reply:
Same comment as https://github.com/puppetlabs/puppet/pull/7619#discussion_r306078172